### ISSUE TITLE
[Quest chain] "Recover the Cargo!" & "Jaina Must Know" & "Survey Alcaz Island" &"Warn Bolvar" each bugged in diff way

### ISSUE THREAD

**Issue by ntrna** [chromiecraft/chromiecraft#1622]
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [X] 30-39
- [ ] 40-49
- [ ] 50-59

### Current Behaviour

"Recover the Cargo" - the questgiver, Renn McGill, does not speak his RP quotes upon completion of the quest (which makes it difficult to follow) -quotes sourced below

"Jaina Must Know" - Jaina's standard greeting text refers in length to someone named "Hendel" as though I brought Hendel to her, but I have never seen nor heard of Hendel

"Survey Alcaz Island" - 1) I was granted credit for surveying the island the moment its' location tag pops up in top center screen "Alcaz Island," although I only afterwards actually flew through the island, surveying it. -video sourced below. 2) The completion text from Jaina indicates that she is going to explain further - she does not, nor does she offer the follow up quest, "Warn Bolvar," although I am appropriately levelled

### Expected Blizzlike Behaviour

Described above and some linked below in source, sorry for not precisely following format

### Source

https://tbc.wowhead.com/npc=23569/renn-mcgill

https://www.youtube.com/watch?v=qZw21NMGRx4 - I recognize this is not the best souce, but it is the oldest I could find

### Steps to reproduce the problem

1. Start the questline in Theramore without having done any quests in the area, other than, in my case, Tethyr & Deserter questlines, & "They Call Him Smiling Jim" (not yet done, "Inspecting the Ruins")

### Extra Notes

idk what rev. hash/commit and modules are

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/595bb6adccbabc714469f3935541978283b8bdfb

### Operating system

Windows 10

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-corldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

**Comment by gurveleg** [comment]
Warn Bolvar! quest was remove din 3.0.2, thats why you can't get it anymore. 
https://wowpedia.fandom.com/wiki/Warn_Bolvar!

Private Hendel refers to the NPC you fight and subdue in a camp, then Jaina and others spawn and take him away.

**Comment by ntrna** [comment]
That's my bad for not researching "Warn Bolvar!" enough.

My point with it and with Renn McGill's missing /say lines was more that these quests seem like they're meant to follow The Missing Diplomat, but because they don't require it beforehand, the lore is jarring & hard to follow. Perhaps this was merely an oversight on Blizz's part back in the day, I don't know.
